token-service/src/main/java/org/p2p/token/service/TokenServiceModule.kt

```diff
@@ -45,13 +45,10 @@ object TokenServiceModule : InjectionModule {
             )
         }
 
-        singleOf(::TokenMetadataInMemoryRepository) bind TokenMetadataLocalRepository::class
-        factoryOf(::TokenMetadataRemoteRepository) bind TokenMetadataRepository::class
-
-        singleOf(::TokenPriceInMemoryRepository) bind TokenPriceLocalRepository::class
-        factoryOf(::TokenPriceRemoteRepository) bind TokenPriceRepository::class
-
-        factoryOf(::TokenServiceRemoteRepository) bind TokenServiceRepository::class
+        single<TokenMetadataLocalRepository> { TokenMetadataInMemoryRepository() }
```

Comment:
and why have you reverted that?

Reply:
I don't know why, but my unit test failed when I using factoryOf(::) or singleOf(::)